shell/platform/fuchsia/flutter/BUILD.gn

```diff
@@ -452,3 +453,134 @@ fuchsia_archive("flutter_runner_tzdata_tests") {
     },
   ]
 }
+
+executable("flutter_runner_scenic_unittests") {
+  testonly = true
+
+  output_name = "flutter_runner_scenic_tests"
+
+  sources = [
+    "component.cc",
+    "component.h",
+    "compositor_context.cc",
+    "compositor_context.h",
+    "engine.cc",
+    "engine.h",
+    "fuchsia_intl.cc",
+    "fuchsia_intl.h",
+    "isolate_configurator.cc",
+    "isolate_configurator.h",
+    "logging.h",
+    "loop.cc",
+    "loop.h",
```

Comment:
loop, task_observers, task_runner_adapter etc are all gone in `master`

Reply:
Done